src/allotropy/parsers/chemometec_nucleoview/nucleoview_reader.py

```diff
@@ -34,5 +34,10 @@ def read(cls, contents: IOType) -> pd.DataFrame:
                 raw_data["Date time"] + raw_data["Time zone offset"]
             )
         raw_data["Sample ID"] = raw_data["Image"].str.split("-", n=3).str[3]
+        raw_data.rename(
+            {"Estimated cell diameter [um]": "Estimated cell diameter (um)"},
+            axis=1,
+            inplace=True,
+        )
```

Comment:
Please avoid using `inplace=True`, it is considered to be potentially harmful and there are even [plans to deprecate it in the future](https://github.com/pandas-dev/pandas/issues/16529).
```suggestion
        raw_data = raw_data.rename(
            {"Estimated cell diameter [um]": "Estimated cell diameter (um)"},
            axis=1,
        )
```